examples/classification/test_MultinomialCNB.py

```diff
@@ -0,0 +1,123 @@
+"""
+================================
+Comparing Complement Naive Bayes to standard Naive Bayes.
+================================
+
+An example showing how MultinomialCNB outperforms MultinomialNB
+on imbalanced text classification tasks.
+
+"""
+print(__doc__)
+import numpy as np
+import random
+
+from nltk.corpus import reuters
```

Comment:
Removed NLTK. I was using the Reuters data set because it's imbalanced, but 20 Newsgroups also demonstrates the superiority of CNB.